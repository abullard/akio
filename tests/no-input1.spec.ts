import { describe, it, expect } from "vitest";
import { spawnWrapper } from "./test-utils";

describe('no-input1.spec.ts', () => {
    /*
        1. search #1 w/input
        2. search #2 w/diff input
        3. --no-input
        4. -i
        5. --no-format
        6. -f  
        7. invalid cli-opts x
        8. invalid command number
    */
    it('should skip prompting user for input, -i', async () => {
        const cmd = "pnpm";
        const args = ['akio', '-i'];

        const { stdout } = await spawnWrapper(cmd, args);

        expect(stdout).toMatchSnapshot();
    });
});
