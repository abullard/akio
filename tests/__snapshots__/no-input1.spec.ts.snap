// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`no-input1.spec.ts > should skip prompting user for input, -i 1`] = `
"
> node ./dist/index.cjs -i

[33mpnpm akio[0m
	-----
ðŸ“¦ [34m@abullard/akio[0m
1. [35mtest      [0m â€” run vitest unit tests
2. [35mtest:stub [0m â€” DO NOT REMOVE - stubbed npm script for snapshot tests
3. [35mbuild     [0m â€” This command builds Akio
4. [35mui        [0m â€” pnpm ui package script passthrough
5. [35mapi       [0m â€” pnpm api package script passthrough

ðŸ“¦ [34mapi[0m
6. [35mapi:test:stubOne[0m â€” DO NOT REMOVE - stubbed npm script for monorepo testing
7. [35mapi:test:stubTwo[0m â€” DO NOT REMOVE - stubbed npm script for monorepo testing
8. [35mapi:test:stubThree[0m â€” DO NOT REMOVE - stubbed npm script for monorepo testing
9. [35mapi:test:stubFour[0m â€” DO NOT REMOVE - stubbed npm script for monorepo testing
10. [35mapi:test:stubFive[0m â€” DO NOT REMOVE - stubbed npm script for monorepo testing
11. [35mapi:test:stubSix[0m â€” DO NOT REMOVE - stubbed npm script for monorepo testing
12. [35mapi:test:stubSeven[0m â€” DO NOT REMOVE - stubbed npm script for monorepo testing
13. [35mapi:test:stubEight[0m â€” DO NOT REMOVE - stubbed npm script for monorepo testing

ðŸ“¦ [34mui[0m
14. [35mui:test:stubOne[0m â€” DO NOT REMOVE - stubbed npm script for monorepo testing
15. [35mui:test:stubTwo[0m â€” DO NOT REMOVE - stubbed npm script for monorepo testing
16. [35mui:test:stubThree[0m â€” DO NOT REMOVE - stubbed npm script for monorepo testing
17. [35mui:test:stubFour[0m â€” DO NOT REMOVE - stubbed npm script for monorepo testing
18. [35mui:test:stubFive[0m â€” DO NOT REMOVE - stubbed npm script for monorepo testing
19. [35mui:test:stubSix[0m â€” DO NOT REMOVE - stubbed npm script for monorepo testing
20. [35mui:test:stubSeven[0m â€” DO NOT REMOVE - stubbed npm script for monorepo testing
21. [35mui:test:stubEight[0m â€” DO NOT REMOVE - stubbed npm script for monorepo testing
"
`;
